libs/covid/data-api/src/lib/modules/websites/websites.controller.spec.ts

```diff
@@ -1,18 +1,32 @@
 import { Test, TestingModule } from '@nestjs/testing';
+import { getRepositoryToken } from '@nestjs/typeorm';
+
+import { Repository } from 'typeorm';
+
+import { County, CountyClaim, CategoryValue, CountyClaimInfo } from '@tamu-gisc/covid/common/entities';
+
 import { WebsitesController } from './websites.controller';
+import { WebsitesService } from './websites.service';
 
 describe('Websites Controller', () => {
   let controller: WebsitesController;
 
   beforeEach(async () => {
     const module: TestingModule = await Test.createTestingModule({
-      controllers: [WebsitesController],
+      providers: [
+        WebsitesService,
+        { provide: getRepositoryToken(CategoryValue), useClass: Repository },
+        { provide: getRepositoryToken(County), useClass: Repository },
+        { provide: getRepositoryToken(CountyClaim), useClass: Repository },
+        { provide: getRepositoryToken(CountyClaimInfo), useClass: Repository }
+      ],
+      controllers: [WebsitesController]
     }).compile();
 
     controller = module.get<WebsitesController>(WebsitesController);
   });
 
   it('should be defined', () => {
-    expect(controller).toBeDefined();
+    expect(controller).toBeDefined;
```

Comment:
This still needs to be a method call, not just a reference.

In other words, missing `()`, like `toBeDefined()`